backend/src/packages/chaiNNer_standard/image_utility/compositing/mix.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+import numpy as np
+
+from nodes.properties.inputs import ImageInput, SliderInput
+from nodes.properties.outputs import ImageOutput
+
+from .. import compositing_group
+
+
+@compositing_group.register(
+    schema_id="chainner:image:mix",
+    name="Mix",
+    description="Mixes 2 images together.",
+    icon="BsLayersHalf",
+    inputs=[
+        ImageInput("Image A"),
+        ImageInput("Image B"),
+        SliderInput(
+            "Mix",
+            minimum=0.0,
+            maximum=1.0,
+            default=0.5,
+            precision=4,
+            controls_step=0.001,
+            scale="linear",
+        ),
+    ],
+    outputs=[
+        ImageOutput(
+            image_type="Input0",
+        )
```

Comment:
This won't work if the images are a different size or have a different number of channels. 

For the number of channels, we could probably automatically add the missing channels. So if you give the node a grayscale and an RGBA image, it will automatically convert the grayscale image to RGBA to make this operation possible. Please use the `as_target_channels` functions to do the conversion.

As for size, we can check that in the image output type:
```suggestion
        ImageOutput(
            image_type="""
                Image {
                    width: Input0.width & Input1.width,
                    height: Input0.height & Input1.height,
                    channels: max(Input0.channels, Input1.channels),
                }
            """,
        ).with_never_reason("Both images must have the same size."),
```